understanding/20/help.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Help</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Users can avoid making mistakes</dd>
-         <dt>Author task</dt><dd>Provide help to users on the function currently being performed</dd>
+         <dt>Goal</dt><dd>Users can avoid making mistakes.</dd>
+         <dt>What to do</dt><dd>Provide help to users on the function currently being performed.</dd>
+         <dt>Why it's important</dt><dd>People with cognitive disabilities can complete their tasks more easily.</dd> 
```

Comment:
@rainbreaw commented
> While this is especially important for individuals with cognitive disabilities, it is really something anyone with a disability may need given that their may be an unexpected barrier between the site, and their abilities or AT. Help is the catch all when all else fails.

Reply:
Since the SC primarily calls out cognitive disabilities, I thought it made sense to highlight. If you have a way of gracefully phrasing this, I'd welcome it, but for now how about
```suggestion
         <dt>Why it's important</dt><dd>People with cognitive or other disabilities can complete their tasks more easily.</dd> 
```

Reply:
@mbgower this works for me, thank you.